src/tools/pontoon/translate.md

```diff
@@ -1,58 +1,50 @@
+This document is a summary to demonstrate the different translation modes available in Pontoon. Other parts of documentation include specific pages for learning how to use Pontoon’s [interface](ui.md) and [translation resources](resources.md).
 
 ### Submit a suggestion
 
-When a translator is in Suggest Mode, or doesn’t have permissions to submit translations directly, a blue `SUGGEST` button will be visible in the lower-right side of the editing space.
-
-![Blue button for suggestions](../../assets/images/pontoon/translate/suggest_button.png)
+When the user is in *Suggestion Mode*, or doesn’t have permissions to submit translations directly, a blue `SUGGEST` button will be displayed in the lower-right side of the editing space.
 
-To suggest a translation, the translator should simply input the translation to the editing space and click `SUGGEST`. Once the translator has suggested the translation, the suggestion will appear in the sidebar. In case of multiple suggestions, sidebar will show the most recent one.
+![Blue button for suggestions](../../assets/images/pontoon/translate/suggest_button.png "Editing space in Pontoon with the blue SUGGEST button")
 
-![Sidebar after suggestion](../../assets/images/pontoon/translate/sidebar_suggest.png)
+To suggest a translation, the user can input the translation in the editor and click `SUGGEST`. The suggestion will then be displayed below the editing space and in the sidebar, where in case of multiple suggestions, only the most recent suggestion will be displayed.
 
-A notification will appear in the window, indicating that your translation has been saved.
+![Sidebar after suggestion](../../assets/images/pontoon/translate/sidebar_suggest.png "Pontoon’s sidebar before and after a suggestion is submitted")
 
-Note that a suggested translation exists only in the Pontoon database, not in the localized file or translation memories.
+Note that a suggested translation exists only in the Pontoon database, not in the repository or translation memories.
 
 #### Manually switch to suggestion mode
 
-Even if a translator has permissions to add translations directly, submitting suggestions to be reviewed helps ensure quality. To manually switch to Suggestion Mode, click the Settings gear to the lower-left side of the editing space and click the icon to the left of `Make Suggestions` so that it becomes a green circle. The `SUGGEST` button should then appear to the lower-right side of the editing space.
-
-![Activate make suggestions](../../assets/images/pontoon/translate/activate_suggest.png)
+Even if the user has permissions to add translations directly, submitting suggestions to be reviewed by another person helps ensure quality. To manually switch to *Suggestion Mode*, click the settings icon in the lower-left side of the editing space and select `Make Suggestions`. The `SUGGEST` button will be displayed in the lower-right side of the editing space, replacing the green `SAVE` button.
```

Comment:
```suggestion
It is best practice to submit suggestions for review by another person to ensure translation quality. Some users with elevated permissions may have their editor set to submit translations directly instead of submitting suggestions for review. To manually switch to *Suggestion Mode*, click the settings icon in the lower-left side of the editing space and select `Make Suggestions`. The `SUGGEST` button will be displayed in the lower-right side of the editing space, replacing the green `SAVE` button.
```
Suggestion to make the sentence fit a bit better considering the ability to translate directly is introduced later in this document.